.github/workflows/tests.yaml

```diff
@@ -93,7 +93,9 @@ jobs:
         run: |
           mypy --ignore-missing-imports --strict src/
 
+      # Do not run on macos until there is a hadolint release for macos (Darwin on arm64 architecture).
       - name: Tox
+        if: matrix.os == 'ubuntu-20.04' || matrix.os == 'ubuntu-22.04' || matrix.os == 'windows-latest'
```

Comment:
instead of not running tox, why don't we not install hadolint in macos until there's a release for Darwin arm64?

Reply:
[e2551bf](https://github.com/sscpac/statick-tooling/pull/31/commits/e2551bf395cdb15e67f38051e9567523b1194270)